src/pybind/mgr/orchestrator.py

```diff
@@ -51,6 +51,12 @@ class OrchestratorValidationError(OrchestratorError):
     """
 
 
+class OrchestratorDeserializeError(OrchestratorError):
```

Comment:
Yeah, previously, I used `orchestrator.OrchestratorValidationError` for that.

Reply:
Change to `orchestrator.OrchestratorValidationError`